### DISCUSSION TITLE
14 febbraio 2023 A - esercizio 2

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#30] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/ce8eb174-8df8-451d-841d-da2bb36230ac)

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/78e2ec1f-edbf-484a-83ab-bdada5c7f55b)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/687b9820-38c6-40f6-ab6f-813704eada47)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/0d4e70cd-d2c7-402a-b2b0-2aa7c6c489f7)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/78519e7f-ae05-4805-aab9-b833186fa7df)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/43faa069-8eee-4b0e-99e0-2d8da1bbc6d3)

**Comment by daromacarlo** [comment]
punto 2a)
![Immagine 2024-02-04 220328](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/0dc0e7de-4bc9-4704-b7d9-a0618c28afec)

**Reply by daromacarlo** [reply]
punto 2b) ![Immagine 2024-02-04 221543](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/e7ae38f6-34fb-49f9-81d5-0fe384777e8c)

**Reply by IonutCicio** [reply]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/c615d258-cee4-4c0b-a133-bfc7d16df602)

**Reply by daromacarlo** [reply]
punto 2c)
![Immagine 2024-02-04 234053](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/56a37795-8324-417c-80d2-aa949c72aea7)
![Immagine 2024-02-04 234109](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/cec6de63-84eb-4e1d-ae19-752ef316a41d)
![Immagine 2024-02-04 234125](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/f088ad37-0b7c-49fa-8928-09016ab467d8)
